src/data-dictionary/events/NrAiIssue/lastModifiedTime.md

```diff
@@ -0,0 +1,9 @@
+---
+name: lastModifiedTime
+type: attribute
+units: seconds (s)
```

Comment:
is this really a unit of 'seconds'? If it's a timestamp it wouldn't be that format. e.g., createTime isn't 'seconds'

Reply:
It's milliseconds ;-)